java/bulk-import/bulk-import-runner/src/main/java/sleeper/bulkimport/job/runner/BulkImportJobDriver.java

```diff
@@ -53,10 +53,9 @@
 import static sleeper.ingest.job.status.IngestJobStartedEvent.validatedIngestJobStarted;
 
 /**
- * This class executes a Spark job that reads in input Parquet files and writes
- * out files of {@link sleeper.core.record.Record}s. This takes a {@link BulkImportJobRunner} implementation,
- * which takes rows from the input files and outputs a file for each Sleeper partition.
- * These will then be used to update the {@link StateStore}.
+ * Executes a Spark job that reads in input Parquet files and writes to a Sleeper table. This takes a
```

Comment:
I think this would read better as `Executes a spark job that reads input Parquet files and writes to a Sleeper table`